package.json

```diff
@@ -23,5 +23,10 @@
   "dependencies": {
     "glob": "^5.0.15",
     "lodash": "^3.10.1"
+  },
+  "devDependencies": {
+    "request" : "^2.67.0",
+    "chai": "^3.4.1",
+    "mocha": "^2.3.4"
```

Comment:
your tests are written in es2015.  If thats the way we are going to go, then we need to update the package.json to include the correct node engine